api/src/main/scala/io/codegalaxy/api/chapter/ChapterWithStatisticsRespData.scala

```diff
@@ -0,0 +1,18 @@
+package io.codegalaxy.api.chapter
+
+import io.codegalaxy.api.stats.StatsData
+import play.api.libs.json._
+
+case class ChapterRespData(chapter: ChapterData)
```

Comment:
If we still use it, I suggest we move this case class with companion object to its own file.

Reply:
Okay